Dockerfiles/agent/amd64/Dockerfile

```diff
@@ -1,4 +1,8 @@
 FROM ubuntu:22.04 AS baseimage
+LABEL baseimage.os=“ubuntu jammy LTS”
+LABEL baseimage.name=“ubuntu:22.04”
+LABEL baseimage.isgbi=“no”
```

Comment:
Do we need this label public?

Reply:
this is for having them defined in the same place as where the base image is configured.